app/schema.py

```diff
@@ -95,6 +117,27 @@ class MenteeUpdate(ExtraForbid):
     in_project_underdog: Optional[bool]
 
 
+class MenteeOptions(ExtraForbid):
```

Comment:
Thank you for completing this, it brings us closer to being able to work with mentee data.